### ISSUE TITLE
Conflict with BlockSlider and setResize in min-height setting

### ISSUE THREAD

**Issue by mayuir2014** [deltanet/adapt_delta_default#3]
There is a conflict in the way that the articlePageSlider and the set-resize.less work together (bearing in mind that the set-resize.less will need to be re-written as an extension).

Currently, if you navigate through the first 2 topics, set-resize works fine. However, if you go through topic 3, and interact with the blockslider component, and then navigate back to any of the other topics, you'll find that on resizing the browser, the currently displayed screen is swallowed up at the bottom and cannot be scrolled into view.

I think the issue is in the min-height setting that both are trying to achieve, as they work indvidually, but not together.

**Comment by dancgray** [comment]
I'm not convinced that it is completely due to the blockslider. The issue is on a page were the blockslider is not rendered and non of the blockslider functions are called. Plus blockslider only affects css min-height for the blockslider article.

I think there is more than one issue here. The theme CSS should not have min-height set on each article as screen-resize.less controls this. Removing min-height from:

https://github.com/deltanet/adapt_delta_default/blob/delta-standards-framework/src/theme/adapt-contrib-vanilla/less/src/theme-extras.less#L220

and on each article makes the problem better but does not eliminate white space below the last trickle locked article. This is likely to be a conflict between min-height settings in screen-resize.less and css height changes in trickle.

Also this function which affects height is called whenever the screen is resized:

https://github.com/deltanet/adapt_delta_default/blob/delta-standards-framework/src/extensions/adapt-contrib-trickle/js/adapt-contrib-trickle.js#L286

Basically to many min-height values being set. We may have to do screen resizing a different way.

**Comment by dancgray** [comment]
Try in blockslider:

			this.stopListening(Adapt, "page:scrollTo", this._onBlockSliderPageScrollTo);
			this.stopListening(Adapt, "page:scrolledTo", this._onBlockSliderPageScrolledTo);

**Comment by RobertPeek** [comment]
The resize functionality has been omitted from the default build until it has been developed as a new extension.
